tests/test_0_utils.py

```diff
@@ -52,3 +58,72 @@ def test_0_3_check_dicom(site_package_path):
         site_package_path, "pydicom/data/test_files/JPEG-lossy.dcm"
     )
     assert is_file_a_dicom(dicom_file)
+
+
+@pytest.mark.utilstest
+def test_0_4_check_make_exec_bin():
+
+    PATH_TO_BIN = "./nekton/bins/dcm2nii"
+    # if bin executable revoke rights
+    if os.access(PATH_TO_BIN, os.X_OK):
+
+        process = subprocess.Popen(
+            ["chmod", "-x", PATH_TO_BIN],
+            stdout=subprocess.PIPE,
+            universal_newlines=True,
+        )
+        process.communicate()
+
+    make_exec_bin()
+    assert os.access(PATH_TO_BIN, os.X_OK)
+
+
+@pytest.mark.utilstest
+def test_0_4_check_make_run_bin(site_package_path):
+    path_dcms = os.path.join(
+        site_package_path, "pydicom/data/test_files/dicomdirtests/98892001/CT5N/"
+    )
+    make_exec_bin()
+    run_bin(path_dcms)
+    out_path = os.path.join(
+        path_dcms, "CT5N_SmartScore_-_Gated_0.5_sec_20010101000000_5.nii.gz"
+    )
+    assert os.path.exists(out_path)
+    os.remove(out_path)
```

Comment:
also cleanup before run_bin, in case it remained from prev run, for some reason.